components/AboutPage/index.js

```diff
@@ -46,18 +46,18 @@ const AboutPage = () => {
 
         <div className={scss.about__section_3_right}>
           <div className={scss.about__section_3_right_mobile}>
-            <img className={scss.about__section_3_right} src="/static/image/about-page/about-image-body-1-mobile.png" />
+            <img className={scss.about__section_3_right} src="/static/image/about-page/about-image-body-1-mobile.png" alt="Group of Suffraggettes"/>
           </div>
 
           <div className={scss.about__section_3_right_desktop}>
-            <img className={scss.about__section_3_right} src="/static/image/about-page/about-image-body-1.png" />
+            <img className={scss.about__section_3_right} src="/static/image/about-page/about-image-body-1.png" alt="Group of Suffraggettes"/>
```

Comment:
I think modern usage has deprecated Suffraggettes in favor of Suffragists for activists in the US.